tests/Plugins/__snapshots__/GitHubEnterprise_EndToEndTest.ambr

```diff
@@ -851,7 +850,7 @@
   │ [31m│[0m ----------                                                                                                       [31m│[0m │
   │ [31m│[0m   1) Visit '<scrubbed-github-url>'<scrubbed-space>[31m│[0m │
   │ [31m│[0m   2) Locate the 'Pull Requests' section                                                                          [31m│[0m │
-  │ [31m│[0m   3) Ensure that 'Allow merge commits -> Default...' is set to 'This is not a valid merge commit message'        [31m│[0m │
+  │ [31m│[0m   3) Ensure that 'Allow merge commits -> Default...' is set to This is not a valid merge commit message          [31m│[0m │
```

Comment:
It looks like we lost the single quotes on this message. And it probably needs a period at the end.